@here/harp-mapview/lib/MapView.ts

```diff
@@ -1866,11 +1866,6 @@ export class MapView extends THREE.EventDispatcher {
         tiltDeg: number = 0,
         azimuthDeg: number = 0
     ): void {
-        if (this.projection.type === ProjectionType.Spherical) {
-            // For globe, the rotation is computed in the tangent space of the geoCenter (in a
-            // "flat sphere" without altitude).
-            this.geoCenter = new GeoCoordinates(target.latitude, target.longitude);
-        }
```

Comment:
Are you sure this is unused? What if I use the lookAt function on globe?

Reply:
On the third line after this block, the variable this.geoCenter is overwritten in any case.

Reply:
achso, I missed that, ok sure